customerrors/error.go

```diff
@@ -18,33 +18,20 @@ package customerrors
 
 import (
 	"fmt"
-	"os"
-
-	"github.com/sonatype-nexus-community/nancy/buildversion"
-	. "github.com/sonatype-nexus-community/nancy/logger"
 )
 
-type SwError struct {
-	Message string
-	Err     error
-}
-
-func (sw SwError) Error() string {
-	return fmt.Sprintf("%s - error: %s", sw.Message, sw.Err.Error())
-}
-
-func (sw SwError) Exit() {
-	os.Exit(3)
+type ErrorExit struct {
+	Message  string
+	Err      error
+	ExitCode int
 }
 
-func Check(err error, message string) {
-	if err != nil {
-		location, _ := LogFileLocation()
-		myErr := SwError{Message: message, Err: err}
-		LogLady.WithField("error", err).Error(message)
-		fmt.Println(myErr.Error())
-		fmt.Printf("For more information, check the log file at %s\n", location)
```

Comment:
I'm not sure exactly where you may have moved this, but this was sorta important. Do you mind linking me to where it's been implemented since this PR is kinda big?

Reply:
It's been replaced by returning `error` and using the customer `ExitError` where a specific exit code needed to bubble up the stack.

Reply:
Right, but where's the part that tells someone to look at their log file for more information?

Reply:
> Right, but where's the part that tells someone to look at their log file for more information?

Ah, I see your point now. I went back through all the places that used to call the old "check" method, and changed them to call a new "constructor" method the prints the old message about where to look in the logs for more info, and returns a new ErrorExit to bubble up the call stack.  
ccc94e1